docs/content/latest/api/ysql/exprs/window_functions/analyzing-a-normal-distribution/_index.md

```diff
@@ -2,7 +2,7 @@
 title: percent_rank(), cume_dist() and ntile() on a normal distribution
 linkTitle: Analyzing a normal distribution
 headerTitle: Analyzing a normal distribution with percent_rank(), cume_dist() and ntile()
-description: Comparing and contrasting percent_rank(), cume_dist() and ntile() on large sets of normally distributed values
+description: Describes an empirical investigation to compare and contrast the window functions percent_rank(), cume_dist() and ntile() on large sets of normally distributed values.
```

Comment:
This one is 167 characters long -- might be OK, but better to shorten it. Something like this, but it doesn't say anything that's not in the "title".

"Compare and contrast the window functions percent_rank(), cume_dist(), and ntile() on large sets of normally distributed values."

Reply:
Done.